src/lib/ecore_win32/ecore_win32_clipboard.c

```diff
@@ -2,10 +2,6 @@
 # include <config.h>
 #endif
 
-#define WIN32_LEAN_AND_MEAN
-#include <windows.h>
-#undef WIN32_LEAN_AND_MEAN
-
```

Comment:
Why removing it?

Reply:
I just don't know if we should remove explicitly includes, as it makes things a little more clear. But I know it will work this way.